2-mandatory/1-create-functions.js

```diff
@@ -3,15 +3,24 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
+function first5(Arr) {
+//let numbers = [ 1, 2, 3, 4, 5, 6, 7]
+let copyOfOriginal = Arr.slice(0, 5); {
+   return copyOfOriginal;
+}
+
+
 }
 
 /*
 Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the same elements, except sorted.
 */
-function sortArray() {
+function sortArray(letters) {
+  let copyOfOriginal = [...letters];
+  copyOfOriginal.sort();
+  return copyOfOriginal;
```

Comment:
This is good. We could also have just directly returned the sorted version like this
```suggestion
function sortArray(letters) {
  let copyOfOriginal = [...letters];
  return copyOfOriginal.sort();
  }
```